server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+const config = require('./webpack.config.js');
+
+const app = express();
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 80.
+app.listen(80, () => {
+  console.log('Example app listening on port 80!\n');
+});
```

Comment:
- [ ] Please remove the `console` code as it is affecting performance and it's not code-related but debugging only 👍 see the hint [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log)